OrbitClientGgp/ClientGgp.cpp

```diff
@@ -76,10 +76,12 @@ bool ClientGgp::RequestStartCapture(ThreadPool* thread_pool) {
     return false;
   }
 
+  // Clean capture data and variables to start from a clean state
+  ClearCapture();
+
   // Start capture
```

Comment:
Not added in this PR, but this comment should also be removed.

Reply:
Done!